### PR TITLE
Russian translation from Crowdin

### PR DESCRIPTION
**Pull request by magul**
Translation made by:
 * 2501 (https://crowdin.com/profile/kirinokirino)
 * Ad0lph (https://crowdin.com/profile/stakansemok)
 * Александр Романович (https://crowdin.com/profile/pandusik666)
 * alex.kurinoy (https://crowdin.com/profile/alex.kurinoy)
 * Alexvinn (https://crowdin.com/profile/Alexvinn)
 * Ali Aliyev (https://crowdin.com/profile/shamkir)
 * Alice K (https://crowdin.com/profile/akuznetsova)
 * Aliya Iskhakova (https://crowdin.com/profile/Aliya_I)
 * Amina Shamil (https://crowdin.com/profile/amina.shamil)
 * an4ik (https://crowdin.com/profile/an4ik)
 * Anatoly (https://crowdin.com/profile/bixnel)
 * Andru1999 (https://crowdin.com/profile/Andru1999)
 * Anna Warzecha (vi) (https://crowdin.com/profile/aniav)
 * Arthur Akela (https://crowdin.com/profile/tretuhin)
 * astrezh (https://crowdin.com/profile/astrezh)
 * Atos Portos (https://crowdin.com/profile/atoskg)
 * Azat Sharafetdin (https://crowdin.com/profile/azat385)
 * Babur MB (https://crowdin.com/profile/BaburMB)
 * BardRandall (https://crowdin.com/profile/BardRandall)
 * boxeus (https://crowdin.com/profile/boxeus)
 * cherry-girl (https://crowdin.com/profile/cherry-girl)
 * Dagar Davletov (https://crowdin.com/profile/dagar)
 * Dima Gudkov (https://crowdin.com/profile/d.o.gudkov)
 * dis-aster (https://crowdin.com/profile/dis-aster)
 * Djendo (https://crowdin.com/profile/Djendo)
 * Dmitry Trofimov (https://crowdin.com/profile/traff)
 * e7k8v12 (https://crowdin.com/profile/e7k8v12)
 * Egor Ushakov (https://crowdin.com/profile/erop)
 * Ekaterina Volkova (https://crowdin.com/profile/catherine-v)
 * Elisa Khudiakova (https://crowdin.com/profile/lisavaeta_k)
 * Елизавета Карпова (https://crowdin.com/profile/cheharda)
 * Garik1303 (https://crowdin.com/profile/pvpgarik)
 * GraceAredel (https://crowdin.com/profile/GraceAredel)
 * Grigoriy Beziuk (https://crowdin.com/profile/gbezyuk)
 * Hwang Igor (https://crowdin.com/profile/i2gor87)
 * Ivan Kadykeev (https://crowdin.com/profile/1407navi)
 * Julia Beresneva (https://crowdin.com/profile/iuliiajune)
 * Katerina Permina (https://crowdin.com/profile/permina.ekaterina)
 * klyudmyla (https://crowdin.com/profile/klyudmyla)
 * Kondorok (https://crowdin.com/profile/Kondorok)
 * Larionov Alexander (https://crowdin.com/profile/DrPeryCox)
 * Maria Kotlyarevskaya (https://crowdin.com/profile/Jasstkn)
 * MaryCherry (https://crowdin.com/profile/MaryCherry)
 * Maksim Shvetsov (https://crowdin.com/profile/maksim2moscow)
 * Mike (https://crowdin.com/profile/insolor)
 * Михаил Хлыстов (https://crowdin.com/profile/Maddy13)
 * monomatana (https://crowdin.com/profile/monomatana)
 * Muxiddin Jumaniyazov (https://crowdin.com/profile/muhiddin_bestboy)
 * Nicolas Smakhtin (https://crowdin.com/profile/nicolas_sm)
 * Olea Gulea (https://crowdin.com/profile/olea_gulea)
 * Oleksandr Redko (https://crowdin.com/profile/redykrom)
 * olgapopova2008 (https://crowdin.com/profile/olgapopova2008)
 * Oxana Beydilli Mirolevich (https://crowdin.com/profile/deniz1983)
 * Oxana Kostikova (https://crowdin.com/profile/Lereena)
 * oxana191 (https://crowdin.com/profile/oxana191)
 * patiencekys (https://crowdin.com/profile/patiencekys)
 * Pavel Karateev (https://crowdin.com/profile/Bannert)
 * Pavel Myshev (https://crowdin.com/profile/PavelMyshev)
 * Родион Карунту (https://crowdin.com/profile/radu.karuntu)
 * polina-ts (https://crowdin.com/profile/polina-ts)
 * qserqsis (https://crowdin.com/profile/qserqsis)
 * Rebecca Poulson (https://crowdin.com/profile/rapoulson)
 * Roman Sverchkov (https://crowdin.com/profile/svrchkv)
 * Ruslan (https://crowdin.com/profile/pycz2007)
 * Rustam Apaev (https://crowdin.com/profile/rustamapaev)
 * sasha-game (https://crowdin.com/profile/sasha-game)
 * Sergey Berebko (https://crowdin.com/profile/serbernar1)
 * Sergey Yanzin (https://crowdin.com/profile/xuthus)
 * Sergo1234 (https://crowdin.com/profile/Sergo1234)
 * Steve Gotchya (https://crowdin.com/profile/steve.bogachev)
 * stikkas17 (https://crowdin.com/profile/stikkas17)
 * Tanya98789 (https://crowdin.com/profile/Tanya98789)
 * Timur (https://crowdin.com/profile/timurchek1997)
 * Vasily (https://crowdin.com/profile/JustAMan)
 * Vika.Roztorguiva (https://crowdin.com/profile/Vika.Roztorguiva)
 * Vitalia (https://crowdin.com/profile/imciflam)
 * Vladimir (https://crowdin.com/profile/vovoka)
 * Vladimir Filonov (https://crowdin.com/profile/vladimir-filonov)
 * Vladislav Grigorev (https://crowdin.com/profile/vladisgrig)
 * Wasilisa Romashkina (https://crowdin.com/profile/wasilisa240)
 * xiva-wgt (https://crowdin.com/profile/xiva-wgt)
 * Yakov2626 (https://crowdin.com/profile/Yakov2626)
 * YuSoldatov (https://crowdin.com/profile/YuSoldatov)

### CODE REVIEW

File: ru/README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 
-## Вступительное слово
+## Добро пожаловать
 
-Добро пожаловать в руководство Django Girls! Мы рады тебя здесь видеть :) В этом руководстве мы заглянем за кулисы веб-технологий и дадим 
-базовое представление о тех вещах, которые собираются в общую картину веба таким, каким мы его знаем.
+Добро пожаловать в учебник Django Girls! Мы рады видеть тебя :) В этом пособии мы заглянем под капот веб-технологий, взглянем на все куски кода, которые потом соберутся в одно целое.
 
-Как и с любой незнакомой темой – это будет приключением. Но не волнуйся, тебе уже хватило смелости оказаться здесь, так что всё будет хорошо :) 
+Как и все неизвестное - это может быть немного страшно. Но не беспокойся, ты уже достаточно преодолела, чтобы зайти сюда :)
```
Comment (telezhnaya) [inline ru/README.md:11]: В предыдущей формулировке это могли читать и мужчины, и женщины, и было ок. Это Django Girls tutorial, и возможно это уместно... Не могу решить, что лучше. Надеюсь кто-то еще откомментирует

Comment (telezhnaya) [reply ru/README.md:11]: читаю дальше, дальше мы обращаемся на вы

Comment (telezhnaya) [inline ru/README.md:11]:
```suggestion
Как и все неизвестное - это может быть немного страшно. Но не беспокойтесь, вы уже достаточно преодолели, чтобы зайти сюда :)
```

File: ru/README.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-> Это руководство распространяется под международной лицензией Creative Commons Attribution-ShareAlike 4.0
-Чтобы ознакомиться с содержанием этой лицензии, посети: https://creativecommons.org/licenses/by-sa/4.0/
+> Это руководство распространяется под международной лицензией Creative Commons Attribution-ShareAlike 4.0. Чтобы ознакомиться с содержанием этой лицензии, посети страницу https://creativecommons.org/licenses/by-sa/4.0/
 
-## Вступительное слово
+## Добро пожаловать
 
-Добро пожаловать в руководство Django Girls! Мы рады тебя здесь видеть :) В этом руководстве мы заглянем за кулисы веб-технологий и дадим 
-базовое представление о тех вещах, которые собираются в общую картину веба таким, каким мы его знаем.
+Добро пожаловать в учебник Django Girls! Мы рады видеть тебя :) В этом пособии мы заглянем под капот веб-технологий, взглянем на все куски кода, которые потом соберутся в одно целое.
```
Comment (telezhnaya) [inline ru/README.md:9]:
```suggestion
Добро пожаловать в учебник Django Girls! Мы рады видеть вас :) В этом пособии мы заглянем под капот веб-технологий, взглянем на все куски кода, которые потом соберутся в одно целое.
```

**Comment by das-g** [comment]
> I have an overall question, we definitely need to make a choice.
> 
> In Russian, you could turn to the person in a more formal or in an informal style.
> If we prefer the informal style, we have to choose masculine or feminine forms for all sentences in past tense. Usually, writers use masculine, but it's Django Girls tutorial, it's not OK in our context. In the other hand, many boys are reading this tutorial, and I am not sure whether it's OK to use only feminine forms everywhere.
> 
> We could also prefer formal style (it's like plural form), no problems with masculine or feminine forms here. But, it sounds much more serious, and we will lose this spirit of talking with a friend.
> 
> Anyway, the text should be rewritten, I started the review, we combine formal and informal forms now, it should be unified. I am ready to make all the needed suggestions after we choose the final variant.
> 
> What do you think about that?

I don't know Russian, so I can't make the final call. But as a guideline, consider that walter2 wrote [in a thread on our Crowdin project](https://crowdin.com/project/django-girls-tutorial/discussions/16/0/home):
>  - When the translation allows a female or male version then the female version of the translation should be chosen. Eg. programmer - Programmiererin

I think that rule can be applied to pronouns, including second-person pronouns, too. You're right that the Django Girls tutorial opts for a more buddy-like tone, so I'd recommend:

- Where "you" is clearly plural in the English original, translate it with the informal second person plural pronoun suitable for females. (Which might be a gender neutral pronoun in Russian, if I understand you correctly.)
- Where "you" is clearly singular in the English original, translate it with the informal second person singular pronoun suitable for a female. (Which would be a female-specific pronoun in Russian, if I understand you correctly.)
- Where unsure whether "you" is singular or plural in the English original, also translate it with the latter pronoun (informal second person singular), unless Russian has an informal second person pronoun for unspecified numerus that could be used instead, which would surprise me.

More generally:

- If you have to choose between different forms of addressing the reader, choose the more/most informal one that isn't derogatory. How friends or other close peers (e.g. siblings) would address each other is a good guideline.
- If that means you have to choose between a male and female form, prefer the female one, even when the target language otherwise generally uses the male form to address people of unknown gender.
- When it's clear whether the reader is addressed individually or the group of all readers is addressed collectively, use the forms with the appropriate numerus (singular or plural, respectively). When in doubt, assume individual addressing and thus use singular.

**Comment by telezhnaya** [comment]
@das-g thanks a lot, I will check and fix all the issues with your instructions

**Comment by magul** [comment]
Hi @telezhnaya, I'm not Russian speaker either. I'm just acting here as a kind of steward, that sync Crowdin project with GitHub repository.

I totally agree with what @das-g wrote (maybe we should codify that in https://github.com/DjangoGirls/translation-guide ?)

@telezhnaya I recall discussion from https://github.com/DjangoGirls/tutorial/pull/1498 but we still don't know how to sync translations from GitHub to Crowdin, so having your improvements applied on Crowdin will (temporary) resolves that problem for us. (otherwise, we face an infinite hell of unsynced improvements and constant Crowdin overwrite of what GitHub contains).

Thanks in advance!

**Comment by telezhnaya** [comment]
@magul just to clarify: do I need to review and make suggestions to this translation here? I think it should be rewritten and I can't approve these changes at the current state.

Sorry, I am not ready to sync suggestions here and in Crowdin. Neither I am not ready to spend time on learning Crowdin right now.

**Comment by magul** [comment]
@telezhnaya 

>  just to clarify: do I need to review and make suggestions to this translation here? I think it should be rewritten and I can't approve these changes at the current state.

I understand the argument about quality drop here. The last thing I would do is decreasing the quality here, so to be clear, I'm not trying to force merge here.

> Sorry, I am not ready to sync suggestions here and in Crowdin. Neither I am not ready to spend time on learning Crowdin right now.

I understand your reluctance to use Crowdin as a new thing (which wasn't said here explicitly but I recall our previous conversation). I personally see advantages of using it (I believe @das-g can also share some thoughts about it, as for last 2-3 years we maintain German translation up-to-date without any extensive workload) and I believe that learning curve there is so flat that everybody can apply changes there in less than 5 minutes.

I can also understand, that you lack time to properly review these changes ensuring their quality, if you wish, you can review them here and I will apply them there, I will also try to find somebody involved in this translation on Crowdin to help here push this (and future) pull request through.

**Comment by telezhnaya** [comment]
@magul 

I am sorry for my absence. The schedule is so tough this summer.

I tried to find hours to continue proofreading - 3 weeks of trying - impossible for me, and it's a massive amount of work, 2-3 hours will not be enough.

I tried to think about how can I delegate this work so that I could be sure of the quality. Anyway, I would want to re-check everything -> looks like a big amount of work again.

I also feel nervous that I generate huge work for you with this synchronization.

As a result: I suggest merging it as it is, without any corrections (you could discard these corrections from my side). Then, let's make Russian part read-only here (in this case, no one will have to move changes by hands). Then, I will promote the Crowdin link and ask people to contribute and proofread the translation.

Pros:
- The repo is synced with Crowdin (we dreamed about that so many times)
- No mechanical work for you
- No sleepless nights for me

Cons:
- For the first period of time, the quality of translation won't be so good, fem-friendly, and grammatically correct. 

I am sure the only negative aspect is not so bad, and we could improve the quality over time.

**Comment by LenaVolzhina** [comment]
Hi everyone!

I also saw this pull request and tried to fit it into my schedule, but I understood that it's not the type of work I want to do. Here are some reasons why.

Firstly, I agree with @telezhnaya that text on GitHub looks better for me right now, and after a fast reading of changes I didn't see any important fixes in it. 

Secondly, these tho versions are inconsistent in female / plural forms, and I personally prefer the GitHub version in this question, so the huge amount of changes in Crowdin will be needed.

And after all, even if we will spend time to review this pull request, and if these changes will be somehow reflected on Crowdin, they still will be not synced, so this problem will arise again and again.

So because of all these reasons I am not ready to review this pull request or encourage, for example, participants of workshops at St Petersburg or Kazan to take part of this process.

From my side of view it seems like optimal decision to find a way to automatically merge the GitHub version to the Crowdin and make GitHub the "master" version - if someone will make changes at Crowdin, then it's necessary to merge this "branch" to GitHub as soon as possible. After all, the tutorial is clearly using the GitHub version.

Thank you for your efforts. I'm sorry that I can't help with this pull request.

**Pull request closed without merging**